Music app/Classes/PresentationLayer/Screens/Main/CollapsedPlayerViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  TestViewController.swift
+//  Music app
+//
+//  Created by Tsaruk Nick on 25.08.23.
+//
+
+import UIKit
+
+class CollapsedPlayerViewController: UIViewController {
+
+    var isPlaying: Bool = false
+    let playButton = UIButton()
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupPlayerUI()
+    }
+    
+    func setupPlayerUI() {
+        view.backgroundColor = .gray
+        // Do any additional setup after loading the view.
```

Comment:
?

Reply:
there is no this file in project maybe another git bug or xcode bug